lib/components/shopping-cart-widget-component.js

```diff
@@ -10,96 +10,40 @@ export default class ShoppingCartWidgetComponent extends BaseContainer {
 		super( driver, by.className( 'cart-toggle-button' ) );
 	}
 
-	open() {
-		let d = webdriver.promise.defer();
+	async open() {
 		const driver = this.driver;
-		const explicitWait = this.explicitWait;
+		const explicitWait = this.explicitWaitMS;
 
-		this.displayedOpen().then(
-			function cartAlreadyOpen() {
-				d.fulfill( true );
-			},
-			function cartClosed() {
-				driverHelper
-					.clickWhenClickable( driver, by.css( '.cart-toggle-button' ), explicitWait )
-					.then(
-						function success() {
-							d.fulfill( true );
-						},
-						function failure() {
-							d.reject( 'Unable to find Shopping Cart Widget button' );
-						}
-					);
-			}
+		await this.displayedOpen();
+		return await driverHelper.clickWhenClickable(
+			driver,
+			by.css( '.cart-toggle-button' ),
+			explicitWait
 		);
-
-		return d.promise;
 	}
 
-	displayedOpen() {
-		let d = webdriver.promise.defer();
-
-		driverHelper.isElementPresent( this.driver, by.css( 'cart-body' ) ).then(
-			function( present ) {
-				if ( present ) {
-					d.fulfill( true );
-				} else {
-					d.reject( 'Cart not open' );
-				}
-			},
-			function() {
-				d.reject( 'Cart not open' );
-			}
-		);
-
-		return d.promise;
+	async displayedOpen() {
+		return await driverHelper.isElementPresent( this.driver, by.css( 'cart-body' ) );
```

Comment:
I'm not sure the fullfill and reject etc. above are needed at all - maybe a separate PR to remove these without moving to await just yet to see if this will just work?!?